index.py

```diff
@@ -35,7 +35,7 @@ def display_page(pathname):
     elif pathname == '/runscanner/sum_over_time':
         return runscanner_yield_over_time_layout
     elif pathname == '/pooling_qc':
-        return pooling_qc_layout
+        return poolqc.pooling_qc_layout
```

Comment:
Doing `import x as y` (line 13) means the function gets renamed to `y` for the duration of the file, so just `pooling_qc_layout`